src/components/calculators/Payslip.vue

```diff
@@ -106,31 +164,164 @@ export default {
         studentLoanLocation: 'england',
         region: 'england'
       },
-      output: {}
+      output: {},
+      payslipResult: false,
+      resultPeriod: '',
+      resultGrossPay: null,
+      checked: ''
+    }
+  },
+  methods: {
+    setResult () {
+      this.resultPeriod = this.input.period
+      this.resultGrossPay = this.input.grossPay
+      this.payslipResult = true
     }
   }
 }
 </script>
 
-<style>
-    .tabs-component {
-        margin: 4em 0;
+<style lang="scss">
+
+  // Start of Flex classes.
+    .flex {
+        display: flex;
     }
 
-    .tabs-component-tabs {
-        border: solid 1px #ddd;
-        border-radius: 6px;
-        margin-bottom: 5px;
+    .flex-wrap {
+      flex-wrap: wrap;
     }
 
-    @media (min-width: 700px) {
-        .tabs-component-tabs {
-            border: 0;
-            align-items: stretch;
+    .flex-grow {
+      flex-grow: 1;
+    }
+
+    // End of Flex classes.
+
+    // Start of Global styles.
+
+    .inline {
+        display: inline-block;
+    }
+
+    .radio-spacing{
+        margin-right: 0;
+        border-radius: 0.2rem;
```

Comment:
If base font size is 16px. .2 rem is not a whole pixel - 3px is .1875rem 4px is .25rem.  px value / 16 = rem